pppd/main.c

```diff
@@ -747,6 +750,108 @@ set_ifunit(int iskey)
     }
 }
 
+/*
+ * net-* scripts to be run come through here.
+ */
+static void main_script(char* script, int wait)
```

Comment:
Should this be named run_net_script instead of main_script? or find an equivalent method that would invoke run_program with the right parameters?

Reply:
Thanks, sure, I didn't see any but I'll look again.  Thanks for the feedback.

Reply:
> Thanks, sure, I didn't see any but I'll look again. Thanks for the feedback.

Rechecked, nothing found.  Did rename as per suggestion.